app/src/main/java/com/example/indoor_volleyball/Fragments/MapsFragment.java

```diff
@@ -0,0 +1,40 @@
+package com.example.indoor_volleyball.Fragments;
+
+import android.os.Bundle;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.fragment.app.Fragment;
+
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.example.indoor_volleyball.R;
+
+
+public class MapsFragment extends Fragment {
```

Comment:
If you haven't built a fragment yet, you don't want to check it in along with everything else, it clutters up your PRs.  You should be able to just uncheck it in your UI to not add it to the commit.